packages/web-components/fast-foundation/src/tree-item/tree-item.template.ts

```diff
@@ -0,0 +1,66 @@
+import { html, ref, when } from "@microsoft/fast-element";
+import { endTemplate, startTemplate } from "../patterns/start-end";
+import { TreeItem } from "./tree-item";
+
+export const TreeItemTemplate = html<TreeItem>`
+    <template
+        role="treeitem"
+        tabindex="${x => (x.disabled ? null : x.focusable ? 0 : -1)}"
+        class="${x => (x.expanded ? "expanded" : "")} ${x =>
+            x.selected ? "selected" : ""} ${x => (x.nested ? "nested" : "")}
+            ${x => (x.disabled ? "disabled" : "")}"
+        aria-expanded="${x => (x.hasItems ? x.expanded : void 0)}"
+        aria-selected="${x => x.selected}"
+        aria-disabled="${x => x.disabled}"
+        @focus=${(x, c) => x.handleFocus(c.event as FocusEvent)}
+        @blur=${(x, c) => x.handleBlur(c.event as FocusEvent)}
+        @keydown=${(x, c) => x.handleKeyDown(c.event as KeyboardEvent)}
+        ${ref("treeItem")}
```

Comment:
Shouldn't need this `ref` as it's the same as `this` inside the component. When you use a `template` like this, it's a standin for the host itself.

Reply:
removed, thanks